RecoLocalMuon/RPCRecHit/plugins/CSCSegtoRPC.cc

```diff
@@ -45,23 +49,35 @@ std::unique_ptr<RPCRecHitCollection> CSCSegtoRPC::thePoints(const CSCSegmentColl
       segmentsInThisEventInTheEndcap++;
     }
 
+    float myTime = -9999.;
+    float myTimeErr = -9999.;
+    int myBx = -99;
+    const float bunchCrossTimeDiff = 25.;  // time between bunch crossings
+
     if (debug)
-      std::cout << "CSC \t loop over all the CSCSegments " << std::endl;
+      LogDebug("CSCSegtoRPC") << "CSC \t loop over all the CSCSegments " << std::endl;
     for (segment = allCSCSegments->begin(); segment != allCSCSegments->end(); ++segment) {
       CSCDetId CSCId = segment->cscDetId();
 
+      myTime = segment->time();
+      myBx = round(myTime / bunchCrossTimeDiff);
+      if (!(myBx <= maxBX && myBx >= minBX))  // rpc read in bx in [-2, 2]
```

Comment:
Are you skipping the BX in [-2, 2]?

Reply:
No, we are skipping the BX outside [-2, 2]

Reply:
> No, we are skipping the BX outside [-2, 2]

Ah, yes, you are right. It was a bit convoluted, though. Maybe simpler
```suggestion
      if ((myBx > maxBX || myBx < minBX))  // rpc read in bx [-2, 2]
```
(but it is not really important)